warehouse/templates/pages/help.html

```diff
@@ -874,7 +874,7 @@ <h3 id="ips">{{ ips() }}</h3>
 
         <h3 id="beta-badge">{{ beta_badge() }}</h3>
         <p>{% trans %}When Warehouse's maintainers are deploying new features, at first we mark them with a small "beta feature" symbol to tell you: this should probably work fine, but it's new and less tested than other site functionality.{% endtrans %}</p>
-        <p>{% trans %}Currently, no features are in beta.{% endtrans %}</p>
+        
```

Comment:
This is removing the actual line from the 'beta badge' section, but the issue is about removing the duplicate line from the 'how to get up addresses' section.

Reply:
Corrected the mistake 👍